### DISCUSSION TITLE
[Feature] compatibility with other synch app

### DISCUSSION THREAD

**Discussion by lanfeuste** [TechbeeAT/jtxBoard#1512] [Ideas]
It would be great to not depend on davx. Is it possible to use other synch app like opensync?

**Comment by patrickunterwegs** [comment]
Hi @lanfeuste , 
 I created jtx Board for DAVx5. I don't intend to adapt any other sync app, basically open sync could also just do it, as the contract is open for all sync apps. 
On the long run I might go a step further and try to offer the sync directly out of jtx Board without a supporting app.
